src/mc2acab/MCNP_ACAB.py

```diff
@@ -86,11 +87,22 @@ def MCNP_ACAB_Mapstar(n):
     '-apypa_verge' : 0.9,
     '-decay_times' : None,
     '-decay_outs' : None,
+    '-sdef' : None,
     '-passive_sector' : None, # so far useless
-    '-nuc_lib': 'EAF', 
-    '-id_Egroup': 'vitJ+', 
+    '-nuc_lib': 'EAF',
+    '-id_Egroup': 'vitJ+',
 }
 def __parse_args(reqs, options, args):
+    if os.path.isfile('logfile.txt'):
```

Comment:
This is not related to the PR topic, remove and we can add it in another commit.

Reply:
OK